README.md

```diff
@@ -83,9 +83,9 @@ Your permissions will be the same between both namespaces, and you will be able
 #### Running KOTS in Helm managed mode
 Steps to run in Helm managed mode:
 1. `okteto pipeline deploy`
-1. Ensure your local context is set to your okteto environment
-1. Set the `IS_HELM_MANAGED` environment variable for the kots deployment `kubectl set env deployment/kotsadm IS_HELM_MANAGED=true`
-1. Remove S3 endpoint: `kubectl set env deployment/kotsadm S3_ENDPOINT=""`
```

Comment:
This is the markdown way of doing numbered lists.  Correct numbers are set during rendering automatically.